professor.py

```diff
@@ -1,24 +1,22 @@
 #! python3
 import socket
-from shared.types import Poll
+from shared.pollTypes import Poll
+import json
 
 
 def prompt_for_poll():
     question = input("Enter your poll question: ")
-    poll = Poll(None, question)
+    poll = Poll(question)
     return poll
 
-
 def prompt_for_ip():
     ip = input("Enter the IP address of the server (eg 192.168.61.1): ")
     port = int(input("Enter the port of the server (eg 1500): "))
     return (ip, port)
 
-
-def send_msg_to_ip(ip, port, msg):
-    clientSocket = socket.socket()
+def send_msg(clientSocket, msg):
     try:
-        clientSocket.connect((ip, port))
+        clientSocket.send(json.dumps(msg).encode())
```

Comment:
We should probably handle connections before we try to send a message.  I moved the connection stuff to the main.